src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/innertypelast/InputInnerTypeLastClassRootClass.java

```diff
@@ -0,0 +1,129 @@
+/*
+* FindBugs - Find Bugs in Java programs
```

Comment:
please simplify this file to keep code that is actually required for test.
All comments should be removed for sure.

Reply:
done